pallets/subtensor/src/staking/remove_stake.rs

```diff
@@ -282,6 +282,7 @@ impl<T: Config> Pallet<T> {
             NetUid::ROOT,
             total_tao_unstaked,
             0, // no fee for restaking
+            true,
```

Comment:
Can be false as it's into root

```suggestion
            false,
```